pkg/tests/apis/openapi_test.go

```diff
@@ -68,15 +68,9 @@ func TestIntegrationOpenAPIs(t *testing.T) {
 	}, {
 		Group:   "iam.grafana.app",
 		Version: "v0alpha1",
-	}, {
-		Group:   "provisioning.grafana.app",
-		Version: "v0alpha1",
```

Comment:
^^ removing provisioning from this list means we generate the API based on the saved snapshot

Reply:
But why'd we do this? Then we can't easily keep it up to date.

Reply:
Our feature branch will still keep it up-to-date.  This lets us have a static spec with all the endpoints that will eventually be there (based on our feature branch)

Once the real backend is merged... this will be returned